.github/workflows/publish-draft-releases.yml

```diff
@@ -323,7 +323,7 @@ jobs:
         if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
         name: fetch and chmod polkadot
         run: |
-          curl -L -o $HOME/.local/bin/polkadot https://github.com/paritytech/polkadot/releases/download/v0.9.16/polkadot
+          curl -L -o $HOME/.local/bin/polkadot https://github.com/paritytech/polkadot/releases/download/v0.9.17/polkadot
```

Comment:
the download link for the polkadot binary is https://github.com/paritytech/polkadot/releases/download/v0.9.17-rc4/polkadot